core/themes/seven/css/style.css

```diff
@@ -1225,21 +1225,22 @@ div.filter-options select {
 .admin-panel .block-title,
 .admin-panel h3 {
   margin: 9px 9px 0;
-  padding: 6px 0 6px 35px; /* LTR */
+  padding: 6px 0;
   font-size: 0.923em;
   text-transform: uppercase;
-  /* All icons derived from Font Awesome. See README.md in the Seven theme.*/
-  background: url('../images/chevron-circle-right--black--64.png') no-repeat left center; /* LTR */
-  background-size: 23px;
 }
-[dir="rtl"] .admin-panel .block-title,
-[dir="rtl"] .admin-panel h3 {
-  margin-left: 0;
-  margin-right: 9px;
-  padding: 6px 35px 6px 0;
-  background: url('../images/chevron-circle-left--black--64.png') no-repeat right center; /* LTR */
-  background-size: 23px;
+.admin-panel .icon svg {
+  width: 24px;
```

Comment:
`em`'s instead?

Reply:
The whole file has now been swapped out for `em` except the base font-size of `#admin-bar`.